pandas/tests/test_strings.py

```diff
@@ -543,10 +543,26 @@ def test_contains(self):
         assert result.dtype == np.bool_
         tm.assert_numpy_array_equal(result, expected)
 
-        # na
-        values = Series(['om', 'foo', np.nan])
-        res = values.str.contains('foo', na="foo")
-        assert res.loc[2] == "foo"
+        # na for category
```

Comment:
can you add the issue number where needed here (maybe just move these to a separate test)

Reply:
@pulkitmaloo can you make this update? I think a new test seems like a good idea.